src/main/resources/plugin.yml

```
@@ -1,4 +1,4 @@
 name: ConstantHunger
 version: @version@
-api-version: 1.13
+api-version: 1.19
```

```suggestion
api-version: 1.13
```
The API version is actually 1.13 on purpose, so that it works on all versions between 1.13 and 1.19. Other than that, looks good!